src/site/stages/build/drupal/graphql/nodeMediaListImages.graphql.js

```diff
@@ -32,6 +32,19 @@ fragment nodeMediaListImages on NodeMediaListImages {
       }
     }
   }
+  fieldTags {
+    entity {
+      ... on ParagraphAudienceTopics {
+        fieldTopics {
+          entity {
+            ... on TaxonomyTermTopics {
+              name
+            }
+          }
+        }
+      }
+    }
+  }
```

Comment:
fysa I think I already added this in https://github.com/department-of-veterans-affairs/vets-website/pull/14556/files#diff-2877ab103207736d4e47278d9ce71d2eae46579a875faacddd786c4413d70108R239